torch/csrc/jit/script/module.h

```diff
@@ -481,6 +482,23 @@ struct Module {
     }
     fn(*this);
   }
+  void train(bool on = true) {
```

Comment:
nit: add a comment above. This ScriptModule is not documented unfortunately (most JIT code isn't; I've tried to tell people in the  past...). This class is user-facing and has public documentation: https://pytorch.org/cppdocs/api/structtorch_1_1jit_1_1script_1_1_module.html#exhale-struct-structtorch-1-1jit-1-1script-1-1-module

Compare with docs of `nn::Module`: https://pytorch.org/cppdocs/api/classtorch_1_1nn_1_1_module.html#exhale-class-classtorch-1-1nn-1-1-module